### PR TITLE
Adjusted the experiment 4 script to fit the new tornadox interface

### PR DESCRIPTION
**Pull request by nathanaelbosch**
Solves #31. I really didn't do anything else or improve anything, but the script should now run as before.

### CODE REVIEW

File: experiments/4_stiff_brusselator.py [hunk 1]
```diff
@@ -55,28 +47,33 @@ def solve_scipy(ivp, tolerance):
...
-        steps = tornado.step.AdaptiveSteps(first_dt, tol, tol)
+        steps = tornadox.step.AdaptiveSteps(abstol=tol, reltol=tol)
 
         # Truncated EK1
         for n in [3, 8]:
-            solver = tornado.ek1.TruncationEK1(
-                num_derivatives=n, ode_dimension=bruss.dimension, steprule=steps
+            solver = tornadox.ek1.TruncationEK1(
+                num_derivatives=n,
+                # ode_dimension=bruss.dimension,
```
Comment (pnkraemer) [inline experiments/4_stiff_brusselator.py]: maybe delete those lines?

File: experiments/4_stiff_brusselator.py [hunk 2]
```diff
@@ -55,28 +47,33 @@ def solve_scipy(ivp, tolerance):
...
             print(
                 f"\tTruncatedEK1 (nu={n}): error={error_pn}, N={num_steps_pn}, t={t_pn} sec (t/N={t_pn/num_steps_pn} sec)"
             )
 
             # Reference EK1
-            solver_dense = tornado.ek1.ReferenceEK1(
-                num_derivatives=n, ode_dimension=bruss.dimension, steprule=steps
+            solver_dense = tornadox.ek1.ReferenceEK1(
+                num_derivatives=n,
+                # ode_dimension=bruss.dimension,
```
Comment (pnkraemer) [inline experiments/4_stiff_brusselator.py]: here as well

**Review by pnkraemer — APPROVED** [review]
Thanks! I left two comments (about deleting a commented out piece of code), but this is not too important. Merge anyway if you feel like it

File: experiments/4_stiff_brusselator.py (hunk 1 above)
Comment (nathanaelbosch) [reply experiments/4_stiff_brusselator.py]: Thanks! I deleted both :+1:

**Pull request merged**